SOURCE/HDF5Output.c

```diff
@@ -770,13 +770,19 @@ void WriteOutputHDF5( grid *mesh, markers *particles, surface *topo, markers* to
         AddFieldToGroup_generic( _TRUE_, name, "Centers" , "z0", 'f', (model.Nx-1)*(model.Nz-1), Cz0, 1 );
     }
 
+    // Add cell flags for debugging 
     AddFieldToGroup_generic( _TRUE_, name, "Centers" , "BCp" , 'c', (model.Nx-1)*(model.Nz-1), mesh->BCp.type, 1 );
-
     AddFieldToGroup_generic( _TRUE_, name, "Flags", "tag_s" , 'c', (model.Nx-0)*(model.Nz-0), mesh->BCg.type,  1 );
     AddFieldToGroup_generic( _TRUE_, name, "Flags", "tag_n" , 'c', (model.Nx-1)*(model.Nz-1), mesh->BCp.type,  1 );
     AddFieldToGroup_generic( _TRUE_, name, "Flags", "tag_u" , 'c', (model.Nx-0)*(model.Nz+1), mesh->BCu.type,  1 );
     AddFieldToGroup_generic( _TRUE_, name, "Flags", "tag_v" , 'c', (model.Nx+1)*(model.Nz-0), mesh->BCv.type,  1 );
 
+    // Add non-iteration log
+    AddFieldToGroup_generic( _TRUE_, name, "Iterations", "rx_abs"          , 'd', Nmodel.nit_max+1, Nmodel.rx_abs         ,  1 );
+    AddFieldToGroup_generic( _TRUE_, name, "Iterations", "rz_abs"          , 'd', Nmodel.nit_max+1, Nmodel.rz_abs         ,  1 );
```

Comment:
What is "dim" argument? I did notice that is always used as 1
Do we need it?

Reply:
Same goes for the 'compress' argument. Do we need it?
First argument should be the most important, but that one looks like an argument, that is used only for development

Reply:
We could need dim at some point, if we decide to store velocity as vectors (dim=2) for example.

We do not need `compress`, I've just removed it.